provider/README.md

```diff
@@ -304,21 +261,17 @@ Unless stated otherwise by the municipality, this endpoint must return only thos
 | `provider_name` | String | Required | The public-facing name of the Provider |
 | `device_id` | UUID | Required | A unique device ID in UUID format |
 | `vehicle_id` | String | Required | The Vehicle Identification Number visible on the vehicle itself |
-| `vehicle_type` | Enum | Required | see [vehicle types](#vehicle-types) table |
-| `propulsion_type` | Enum[] | Required | Array of [propulsion types](#propulsion-types); allows multiple values |
-| `event_type` | Enum | Required | See [event types](#event-types) table |
-| `event_type_reason` | Enum | Required | Reason for status change, allowable values determined by [`event type`](#event-types) |
-| `event_time` | [timestamp][ts] | Required | Date/time that event occurred at. See [Event Times](#event-times) |
+| `vehicle_type` | Enum | Required | see [vehicle types][vehicle-types] table |
+| `propulsion_type` | Enum[] | Required | Array of [propulsion types][propulsion-types]; allows multiple values |
+| `vehicle_state` | Enum | Required | See [vehicle state][vehicle-states] table |
+| `event_type` | Enum[] | Required | [Vehicle event(s)][vehicle-events] for state change, allowable values determined by `vehicle_state`. |
```

Comment:
Question: we left this field as `event_type` although it is now an array - this aligns with how we treat `propulsion_type` so I'm not so worried. But in `/vehicles` we call it `last_event_types`. Should we use the plural here, or the singular there?

Reply:
We'll resolve this as part of #526